### PR TITLE
Evaluations ui move results to bottom

### PR DESCRIPTION
**Pull request by johannkm**
Update, I turned down the height of the result box a bit. Not sure which I prefer
Screenshot 2023-07-19 at 11 51 44 PM

Old images with a larger box
Screenshot 2023-07-19 at 5 38 26 PM
Screenshot 2023-07-19 at 5 38 20 PM
Screenshot 2023-07-19 at 5 38 17 PM
Screenshot 2023-07-19 at 5 41 11 PM

### CODE REVIEW

**Comment by johannkm** [comment]
Current dependencies on/for this PR:
* master
  * **PR #15200** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/15200" target="_blank">Graphite</a> 
    * **PR #15204** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/15204" target="_blank">Graphite</a> 
      * **PR #15401** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/15401" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/15401?utm_source=stack-comment).

File: js_modules/dagit/packages/core/src/assets/AutoMaterializePolicyPage/AutomaterializeMiddlePanelNoPartitions.tsx [hunk 1]
```diff
@@ -57,14 +57,28 @@ export const AutomaterializeMiddlePanelNoPartitions = ({
...
+        // style={{flex: '0 0 96px'}}
+        border={{side: 'bottom', width: 1, color: Colors.KeylineGray}}
+        flex={{alignItems: 'center', justifyContent: 'space-between'}}
+      >
+        <Box
+          style={{flex: '0', backgroundColor: Colors.Gray100}}
+          padding={{horizontal: 32, vertical: 24}}
+        >
+          <Subheading>Result</Subheading>
+        </Box>
```
Comment (hellendag) [inline js_modules/dagit/packages/core/src/assets/AutoMaterializePolicyPage/AutomaterializeMiddlePanelNoPartitions.tsx:77]: This style seems very unusual to me. Are there other places in the app where we use a padded gray box this way?

**Comment by johannkm** [comment]
gentle ping @braunjj

**Comment by braunjj** [comment]
Hey guys, I put together an updated design for this that you can find in figma [here](https://www.figma.com/file/hl3Ik2p3yOR0R21dHLbn5t/Auto-materialize-Policy-Evaluation?type=design&node-id=309-16622&mode=design&t=WxT29AZAAc68PrMn-4).

image
image
image

**Comment by johannkm** [comment]
Design from Josh

image (5)

**Comment by johannkm** [comment]
Closing because I haven't had time to implement and the current UI is probably good enough. I hope we do this at some point though

**Pull request closed without merging**